tests/congruence_tests/test_query.py

```diff
@@ -1,4 +1,4 @@
-from typing import Tuple, Callable, Any, Union
+from typing import Tuple, Callable, Any
```

Comment:
`Tuple` has been depracated in favor of bultin `tuple` since 3.9. This might be a good chance to drop those too. See also https://docs.python.org/3/library/typing.html#deprecated-aliases

Reply:
wondering how I missed all that...
thanks